### ISSUE TITLE
Text tool tip can be placed far outside the flutter view or disappear entirely when changing orientation

### ISSUE THREAD

**Issue by robertjobin** [flutter/flutter#50582]
## Steps to Reproduce

1. Launch a flutter view with a text box
2. Enter a few characters of text
3. Long press to select text
4. Rotate your test device from portrait to landscape
5. Observe tool tip appears outside of the normal bounds of the flutter view
6. Change orientation back to portrait
7. Observe tool tip is now gone
8. Long press text again
9. Observe tool tip is once again visible

**Expected results:** 

The tool tip should remain within the bounds of the flutter view and should always be visible after orientation changes.

**Actual results:** 

The tool tip text can either appear in unexpected places or disappear entirely

This information will be provided by our dev team:
<details>
  <summary>Logs</summary>

```
```

```
```

```
```

</details>

**Comment by VladyslavBondarenko** [comment]
After switching from portrait to landscape, true for material

![image](https://user-images.githubusercontent.com/17024977/74317717-9475ee80-4d84-11ea-8cd3-0e04dc662143.png)

But I can't make it disappeared

<details>
 <summary>Code</summary>

```
import 'package:flutter/material.dart';

void main() => runApp(Home());

class Home extends StatefulWidget {
  _HomeState createState() => _HomeState();
}

class _HomeState extends State<Home> {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
          body: Center(
            child: TextField(),
          ),
      ),
    );
  }
}
```

</details>

**Comment by dannyvalentesonos** [comment]
We closed this bug because it no longer happens now. I think it was resolved with the nested overlay bug fixes from: #43787
